tests/lib.rs

```diff
@@ -229,44 +233,66 @@ fn assert_call_number(test: &str, expected_calls: usize) {
 
 #[test]
 fn test_fdr_control1() {
-    control_fdr("test_fdr_ev_1", "SOMATIC", 0.05, false);
+    control_fdr("test_fdr_ev_1", vec!["SOMATIC"], 0.05, false);
     //assert_call_number("test_fdr_ev_1", 974);
 }
 
 #[test]
 fn test_fdr_control2() {
-    control_fdr("test_fdr_ev_2", "SOMATIC", 0.05, false);
+    control_fdr("test_fdr_ev_2", vec!["SOMATIC"], 0.05, false);
     assert_call_number("test_fdr_ev_2", 985);
 }
 
 /// same test, but low alpha
 #[test]
 fn test_fdr_control3() {
-    control_fdr("test_fdr_ev_3", "ABSENT", 0.001, false);
+    control_fdr("test_fdr_ev_3", vec!["ABSENT"], 0.001, false);
     assert_call_number("test_fdr_ev_3", 0);
 }
 
 #[test]
 fn test_fdr_control4() {
-    control_fdr("test_fdr_ev_4", "SOMATIC_TUMOR", 0.05, false);
+    control_fdr("test_fdr_ev_4", vec!["SOMATIC_TUMOR"], 0.05, false);
     assert_call_number("test_fdr_ev_4", 0);
 }
 
 #[test]
 fn test_fdr_control_local1() {
-    control_fdr("test_fdr_local1", "SOMATIC", 0.05, true);
+    control_fdr("test_fdr_local1", vec!["SOMATIC"], 0.05, true);
     assert_call_number("test_fdr_local1", 0);
 }
 
 #[test]
 fn test_fdr_control_local2() {
-    control_fdr("test_fdr_local2", "SOMATIC", 0.25, true);
+    control_fdr("test_fdr_local2", vec!["SOMATIC"], 0.25, true);
     assert_call_number("test_fdr_local2", 1);
 }
 
+#[test]
+fn test_fdr_control_local3() {
+    control_fdr(
+        "test_fdr_local3",
+        vec![
+            "GERMLINE",
+            "SOMATIC_TUMOR_HIGH",
+            "SOMATIC_TUMOR_LOW",
+            "ARTIFACT",
+            "FFPE_ARTIFACT",
```

Comment:
Why do you now give all the events here? Just GERMLINE and SOMATIC_TUMOR_LOW, like in the config.